reactjs-films-homework/src/languages/languagesData.ts

```diff
@@ -32,7 +36,12 @@ const data = {
     showAll: 'Show all',
     titleName: 'Title',
     min: 'min',
+    birthday: 'Birthday',
+    placeOfBirth: 'Place of birth',
+    biography: 'Biography',
+    photos: 'Photos',
   },
 };
```

Comment:
After fulfilling general requirements you can check libs for localization and apply one of them to your project.
For example:
https://github.com/i18next/react-i18next